Source/DafnyLanguageServer/Language/Symbols/DafnyLangSymbolResolver.cs

```diff
@@ -150,9 +154,60 @@ private class SymbolDeclarationResolver {
           cancellationToken.ThrowIfCancellationRequested();
           functionSymbol.Parameters.Add(ProcessFormal(scope, parameter));
         }
+        if (function.Result != null) {
+          functionSymbol.Parameters.Add(ProcessFormal(scope, function.Result));
```

Comment:
Do you have an example where this is necessary?

Reply:
I'm adding a new test but yes:
```
function f(i: int): (r: int)
  ensures r - i < 10
{
  i + 2
}
```
Without this change, hovering the "r" will not display any information, and it won't be possible to "jump to definition"